src/ui/pages/ErrorsPage.tsx

```diff
@@ -30,7 +31,7 @@ export default function ErrorsPage() {
 
     // Loaded plugin errors
     errors.concat(Object.values(plugins).filter(plugin => plugin.errors.length).map(plugin => ({
-        header: `${plugin.name} v${plugin.manifest.version} had an error.`,
+        header: `${plugin.name} v${plugin.manifest.version} errored.`,
```

Comment:
```suggestion
        header: `${plugin.name} v${plugin.manifest.version} experienced an error.`,
```

Reply:
Plugins can have multiple errors in a card, so I just thought if putting "an error" would suit. We could put the errors in different cards, but I'm just afraid it would be spammy